app/services/rummager_link_sync.rb

```diff
@@ -0,0 +1,27 @@
+class RummagerLinkSync
```

Comment:
The simplest way of thinking about instance/class method choice is if there's a state then consider it to be instance, if not consider it to be a class method.